app/models/facet_collection.rb

```diff
@@ -20,6 +20,19 @@ def filters
     facets.select(&:filterable?)
   end
 
+  def ensure_minimum_one_open_facet(facets_to_ensure)
+    unless at_least_one_facet_selected? facets_to_ensure
+      if defined? facets_to_ensure.first.open_facet!
+        facets_to_ensure.first.open_facet!
+      end
+    end
+    facets_to_ensure
+  end
+
+  def at_least_one_facet_selected?(facets_to_ensure)
```

Comment:
I'm not sure about the name `facets_to_ensure`.  Is there something simpler it could be called, e.g. just `facets` if this variable name hasn't already been used?